### ISSUE TITLE
[BUG] convertAlleleCounts.r returns logR = 0 when no coverage in tumour sample

### ISSUE THREAD

**Issue by jherrero** [nf-core/sarek#496]
## Description of the bug

convertAlleleCounts.r returns a logR or 0 when tumourcounts is 0.

Line 85 aims at catching log(0/0) cases, but in line 73, any log(n/0) or log(0/n) that should return +/- Infinity are changed to NA. As a result cases where there is coverage in normal only or in tumour only end up with an incorrect logR = 0.

## Additional context

Note that current version of ASCAT ignores non-finite numbers. As such, returning NA or -Infinity doesn't matter at this time, although -Inf gives ASCAT the option to capture these cases in a future version. Alternatively, convertAlleleCounts.r could add a small delta to both tumour and normal counts to avoid +/- Infinity.

In short, there is no perfect solution, at least for now, but returning logR =0 signals lack of CNA instead of, for instance, homozygous deletion. Keeping these logR as -Infinity/+Infinity or simply NA (as for the BAF) would be preferable.

**Comment by FriederikeHanssen** [comment]
Thank you for reporting this. Ping @lassefolkersen , not sure if this could still be a problem in the module

**Comment by maxulysse** [comment]
We won't be converting allecounts in the coming release if I'm not mistaken since the new ascat can handle bam inputs, so won't be an issue anymore indeed.

**Comment by lassefolkersen** [comment]
Yes, I think so too @maxulysse - the entire thing have (now) been delegated to internal ASCAT 3.0 functions. First the `ascat.prepareHTS` function - using the official `alleleCounter` image (mulled with the official bioconda ASCAT-3.0 image), and then the `ascat.loadData` function to execute the actual ASCAT functions. I would like to test this, however, @jherrero would you like to help with that? It's basically just a matter of running the same setup that you report, but using  the nf-dsl2 module setup in this [pull request](https://github.com/nf-core/modules/pull/1332/files).

**Comment by FriederikeHanssen** [comment]
@SusiJo is working on this by adding ASSCAT 3.0 into the pipeline

**Comment by FriederikeHanssen** [comment]
Closed by #606 , upgrade to ASCAT 3.0 makes this script obsolet, so this should fix the issue.
